### PR TITLE
Try fix `01290_max_execution_speed_distributed`

### PR DESCRIPTION
**Pull request by nickitat**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Related: #71366

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
*This is an automated comment for commit e17973130d6e52f8bcffcfacaf0da8691dc32f76 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/72951/e17973130d6e52f8bcffcfacaf0da8691dc32f76/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/builds/report.html">✅ success</a></td></tr>
<tr><td>ClickBench</td><td>Runs [ClickBench](https://github.com/ClickHouse/ClickBench/) with instant-attach table</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/clickbench__aarch64_.html">✅ success</a></td></tr>
<tr><td>Compatibility check</td><td>Checks that <b>clickhouse</b> binary runs on distributions with old libc versions. If it fails, ask a maintainer for help</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/compatibility_check__aarch64_.html">✅ success</a></td></tr>
<tr><td>Docker keeper image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/docker_keeper_image.html">✅ success</a></td></tr>
<tr><td>Docker server image</td><td>The check to build and optionally push the mentioned image to docker hub</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/docker_server_image.html">✅ success</a></td></tr>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td>✅ success</td></tr>
<tr><td>Install packages</td><td>Checks that the built packages are installable in a clear environment</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/install_packages__aarch64_.html">✅ success</a></td></tr>
<tr><td>Integration tests</td><td>The integration tests report. In parenthesis the package type is given, and in square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/integration_tests__aarch64__[1_6].html">✅ success</a></td></tr>
<tr><td>Performance Comparison</td><td>Measure changes in query performance. The performance test report is described in detail <a href="https://github.com/ClickHouse/ClickHouse/tree/master/docker/test/performance-comparison#how-to-read-the-report">here</a>. In square brackets are the optional part/total tests</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/performance_comparison__release__[1_4]/report.html">✅ success</a></td></tr>
<tr><td>Stateful tests</td><td>Runs stateful functional tests for ClickHouse binaries built in various configurations -- release, debug, with sanitizers, etc</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/stateful_tests__aarch64_.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/419544067f3a7323e6649817f02840b6db7aa538/unit_tests__asan_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: tests/queries/0_stateless/01290_max_execution_speed_distributed.sql [hunk 1]
```diff
@@ -21,6 +21,7 @@ FROM system.query_log
 WHERE
     current_database = currentDatabase() AND
     event_date >= yesterday() AND
+    event_time >= now() - INTERVAL '5 MINUTES' AND -- time limit for tests not marked `long` is 3 minutes, 5 should be more than enough
```
Comment (Algunenano) [inline tests/queries/0_stateless/01290_max_execution_speed_distributed.sql:24]: Maybe we could just reset the settings to their default values. Or just set them only in the query we care about

Comment (nickitat) [reply tests/queries/0_stateless/01290_max_execution_speed_distributed.sql:24]: imo we shouldn't read 869678 rows from `system.query_log` anyway, but leave global settings with default values also wouldn't harm 👍🏼

**Review by Algunenano — APPROVED** [review]

**Review by vdimir — APPROVED** [review]
So, was it timing out on the `system.query_log` query but not on the query we tested? Great catch, and it’s a pity that I didn’t notice it initially

**Comment by nickitat** [comment]
> So, was it timing out on the `system.query_log` query but not on the query we tested? Great catch, and it’s a pity that I didn’t notice it initially

that was my impression because in my pr it failed on the query_log. but flaky check proves me wrong. let's reduce number of rows to read for the main query too.

**Comment by nickitat** [comment]
`Stateless tests (aarch64, asan) [1/2]` - https://github.com/ClickHouse/ClickHouse/issues/72922
`Stateless tests (debug, s3 storage)` - https://github.com/ClickHouse/ClickHouse/pull/72890

**Pull request merged**